src/core/agent/services/signifyNotificationService.ts

```diff
@@ -102,6 +102,28 @@ class SignifyNotificationService extends AgentService {
         notifications.notes.shift();
       }
       for (const notif of notifications.notes) {
+        await this.markNotification(notif.i);
+        if (notif.a.r === NotificationRoute.MultiSigIcp) {
+          const multisigNotification = await this.signifyClient
+            .groups()
+            .getRequest(notif.a.d);
+          if (!multisigNotification || !multisigNotification.length) {
+            continue;
+          }
+          const multisigId = multisigNotification[0]?.exn?.a?.gid;
+          if (!multisigId) {
+            await this.markNotification(notif.i);
+            continue;
+          }
+          const hasMultisig = await Agent.agent.multiSigs.hasMultisig(
+            multisigId
+          );
+          const notificationsForThisMultisig =
+            await this.findNotificationsByMultisigId(multisigId);
+          if (hasMultisig || notificationsForThisMultisig.length) {
+            continue;
```

Comment:
We should be marking here. In the future, we will do some processing on this notification (e.g. to know how many people accepted the inception event etc)